app/controllers/staff/events_controller.rb

```diff
@@ -22,6 +22,16 @@ def speaker_emails
   def guidelines
   end
 
+  def update_guidelines
+    if @event.update_attributes(event_params)
```

Comment:
This is more of a nitpick but worth point out.  This update_attributes is going to accept and update other values on the event beyond guidelines.  If we allow program team members to update guidelines, which might happen, then this method would allow them to slip through additional attributes. 

Additionally, you can also just call @event.update(params) as a shortened version of update_attributes.  It does the same thing.